java-components/domainproxy/server/src/main/java/com/redhat/hacbs/domainproxy/GAV.java

```diff
@@ -0,0 +1,4 @@
+package com.redhat.hacbs.domainproxy;
+
+public record GAV(String group, String artifact, String version) {
+}
```

Comment:
While I realise this was part of this codebase this will make the 4th GAV class in JBS (!)

Reply:
Should definitely be removed at some point, agreed!